release/release.mk

```diff
@@ -49,3 +49,23 @@ sign-keyless-release: sign-keyless-cosign-release sign-keyless-cosigned-release
 .PHONY: snapshot
 snapshot:
 	LDFLAGS="$(LDFLAGS)" goreleaser release --skip-sign --skip-publish --snapshot --rm-dist --timeout 60m
+
+####################
+# copy image to GHCR
+####################
+
+.PHONY: copy-cosign-signed-release-to-ghcr
+copy-cosign-signed-release-to-ghcr:
+	cosign copy $(KO_PREFIX)/cosign:$(GIT_VERSION) $(GHCR_PREFIX)/cosign:$(GIT_VERSION)
```

Comment:
right now we are pushing the ci images to the following paths:
- `ghcr.io/sigstore/cosign/cosign` for cosign
- `ghcr.io/sigstore/cosign/cosigned` for cosigned
- `ghcr.io/sigstore/cosign/sget` for sget

I think for now we might keeping push to those paths

I've copy the gcr image of v.1.6.0 today to the `ghcr.io/sigstore/cosign/cosign`

```
$ COSIGN_EXPERIMENTAL=1 cosign verify ghcr.io/sigstore/cosign/cosign:v1.6.0

Verification for ghcr.io/sigstore/cosign/cosign:v1.6.0 --
The following checks were performed on each of these signatures:
  - The cosign claims were validated
  - Existence of the claims in the transparency log was verified offline
  - Any certificates were verified against the Fulcio roots.

[{"critical":{"identity":{"docker-reference":"gcr.io/projectsigstore/cosign"},"image":{"docker-manifest-digest":"sha256:b667002156c4bf9fedd9273f689b800bb5c341660e710e3bbac981c9795423d9"},"type":"cosign container image signature"},"optional":{"Bundle":{"SignedEntryTimestamp":"MEUCIQDKOjHLFQDrOfI0FGxaOUVcrvuh639SwV+4rhim2cg3ZAIgctpg49VMRpvKJ5ENfLuma6vcfaoxaWa6i8GaRhF/HLo=","Payload":{"body":"eyJhcGlWZXJzaW9uIjoiMC4wLjEiLCJraW5kIjoiaGFzaGVkcmVrb3JkIiwic3BlYyI6eyJkYXRhIjp7Imhhc2giOnsiYWxnb3JpdGhtIjoic2hhMjU2IiwidmFsdWUiOiI2ZDI3N2QyMWNlNDJmODgzNjM0ZGYyMTM5MzhjNGUxOGYzNTI0N2I5OGZiYmZlY2ExNzY1MWE1MjQ1MjIxYmEwIn19LCJzaWduYXR1cmUiOnsiY29udGVudCI6Ik1FVUNJUUNMOEJFKzdmeWJyNjZDK1RlNEs4NTBoNEFmb2dEand3WkhFaEtYQjkyL3RRSWdjS0luQkVkWHFoWHBYeDJWRFVjZmxwOUMxdlFrQXUwZHRIczdadEYzd213PSIsInB1YmxpY0tleSI6eyJjb250ZW50IjoiTFMwdExTMUNSVWRKVGlCRFJWSlVTVVpKUTBGVVJTMHRMUzB0Q2sxSlNVTk1WRU5EUVdKTFowRjNTVUpCWjBsVVkwdDNURWxSTUVGdGVISlZTRlZtVkhKaWJDOW9aM05YZEdwQlMwSm5aM0ZvYTJwUFVGRlJSRUY2UVhFS1RWSlZkMFYzV1VSV1VWRkxSWGQ0ZW1GWFpIcGtSemw1V2xNMWExcFlXWGhGVkVGUVFtZE9Wa0pCVFZSRFNFNXdXak5PTUdJelNteE5RalJZUkZSSmVRcE5SRTEzVGtSQk5FMVVTVEJQVm05WVJGUkplVTFFVFhkT1JFRTBUV3BKTUU5R2IzZEJSRUphVFVKTlIwSjVjVWRUVFRRNVFXZEZSME5EY1VkVFRUUTVDa0YzUlVoQk1FbEJRa2huVDBsSmJGUkRMMUpQUW1kVFNtbG9VMkZxYlRoVGNrdGtSRmcyYXk5a2VXZzFVMHRoYTNCWlVUSkxUR0ZUZFd3eGRrSTFMeThLVEVkVk1pOUlTM0JtZFV4VWRqZ3ZUaXRCTTI1R1lrVmhTakp1YlRGaFpXcG5aVUYzWjJRd2QwUm5XVVJXVWpCUVFWRklMMEpCVVVSQloyVkJUVUpOUndwQk1WVmtTbEZSVFUxQmIwZERRM05IUVZGVlJrSjNUVVJOUVhkSFFURlZaRVYzUlVJdmQxRkRUVUZCZDBoUldVUldVakJQUWtKWlJVWkNZVE5XTTBOb0NrWkZZbk01V214SVowNTBhRlUwV1ZVeVlrUjBUVUk0UjBFeFZXUkpkMUZaVFVKaFFVWkdha0ZJYkN0U1VtRldiWEZZY2sxclMwZFVTWFJCY1hoaldEWUtUVVF3UjBFeFZXUkZVVVZDTDNkUmVrMUVSMEpNTW5Sc1pWZDRiR016VGtGalNFcDJZVzFXYW1SSVRuQmFNMDR3WWpOS2JFeHRiR2hpVXpWdVl6SldlUXBrYld4cVdsZEdhbGt5T1RGaWJsRjFXVEk1ZEUxRGEwZERhWE5IUVZGUlFtYzNPSGRCVVVWRlJ6Sm9NR1JJUW5wUGFUaDJXVmRPYW1JelZuVmtTRTExQ2xveU9YWmFNbmhzVEcxT2RtSlVRVXRDWjJkeGFHdHFUMUJSVVVSQmQwNXdRVVJDYlVGcVJVRnROV013UWtSYVdVOXpNMDByZGxRd01DdDFXbEJXZDJnS1RHNXdORXB5TWs0dmFXTnpPV0ZLWXk5UFNrb3ZRa1JIWTIwMVMzRnFTVkYzZDFVeVR6UnBaRUZxUlVFNU9FeHlXR3RhUlhoaE1UWlFSM2t6VGxOVlJBcEZkakZpVUhGNU5tbzBaRkZCUzBzM1dXOVlXRlpNY0hkbU16SjBaSE5aWW14aFFYQnlVakZ0Y2sxTGJ3b3RMUzB0TFVWT1JDQkRSVkpVU1VaSlEwRlVSUzB0TFMwdENnPT0ifX19fQ==","integratedTime":1646381571,"logIndex":1556936,"logID":"c0d23d6ad406973f9559f3ba2d1ca01f84147d8ffc5b8445c224f98b9591801d"}},"GIT_HASH":"4b2c3c0c8ee97f31b9dac3859b40e0a48b8648ee","GIT_VERSION":"v1.6.0","Issuer":"https://accounts.google.com","Subject":"keyless@projectsigstore.iam.gserviceaccount.com"}}]
```

Reply:
thanks for pointing out the current ghcr prefix.
changed it to ghcr.io/sigstore/cosign, to match the current behavior.

is this what we want for the other repos as well, ghcr.io/sigstore/fulcio/fulcio and ghcr.io/sigstore/rekor/rekor ?

Reply:
for fulcio we can keep `ghcr.io/sigstore/fulcio`
but for rekor we might do the similar from cosign because we have both rekor-server and cli

```
ghcr.io/sigstore/rekor/rekor-server
ghcr.io/sigstore/rekor/rekor-cli
```